pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -699,7 +704,12 @@ func (k *Kubernetes) Undeploy(komposeObject kobject.KomposeObject, opt kobject.C
 		return errors.Wrap(err, "k.Transform failed")
 	}
 
-	client, namespace, err := k.GetKubernetesClient()
+	client, ns, err := k.GetKubernetesClient()
+	namespace := ns
+	if opt.IsNamespaceFlag {
+		namespace = opt.Namespace
+	}
+
```

Comment:
@kadel I had added it for `down` earlier but the issue I faced was, if we run `kompose down` and there is nothing deployed in `kompose` will still show 
```bash
INFO[0000] Deleting application in "default" namespace
```